client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -38,31 +39,22 @@ function checkPropsChange( nextProps, propArr ) {
 	return false;
 }
 
-export default React.createClass( {
-	displayName: 'Plugins-list-header',
-
-	propTypes: {
-		label: React.PropTypes.string,
-		isBulkManagementActive: React.PropTypes.bool,
-		toggleBulkManagement: React.PropTypes.func.isRequired,
-		updateAllPlugins: React.PropTypes.func.isRequired,
-		updateSelected: React.PropTypes.func.isRequired,
-		haveUpdatesSelected: React.PropTypes.bool,
-		pluginUpdateCount: React.PropTypes.number.isRequired,
-		activateSelected: React.PropTypes.func.isRequired,
-		deactiveAndDisconnectSelected: React.PropTypes.func.isRequired,
-		deactivateSelected: React.PropTypes.func.isRequired,
-		setAutoupdateSelected: React.PropTypes.func.isRequired,
-		setSelectionState: React.PropTypes.func.isRequired,
-		unsetAutoupdateSelected: React.PropTypes.func.isRequired,
-		removePluginNotice: React.PropTypes.func.isRequired,
-		haveActiveSelected: React.PropTypes.bool,
-		haveInactiveSelected: React.PropTypes.bool,
-		bulkManagement: React.PropTypes.bool,
-		selectedSiteSlug: React.PropTypes.string,
-		plugins: React.PropTypes.array.isRequired,
-		selected: React.PropTypes.array.isRequired
-	},
+export class PluginsListHeader extends Component {
+
+	constructor( props ) {
+		super( props );
+		this.state = {
```

Comment:
Also, `this.state` can be defined as a class property like this:

```
state = {
	...
};
```

which will help you avoid adding a `constructor()` method.